### PR TITLE
Add per-host image setting

### PR DESCRIPTION
**Pull request by moio**
## What does this PR change?

Adds a setting to change the image per host (on hosts for which it makes sense to configure an image)

### CODE REVIEW

File: modules/cucumber_testsuite/main.tf [hunk 1]
```diff
@@ -30,12 +29,15 @@ locals {
...
   host_key => lookup(var.host_settings[host_key], "additional_repos", {}) if var.host_settings[host_key] != null }
+  images = { for host_key in local.hosts :
+  host_key => lookup(var.host_settings[host_key], "image", null) if var.host_settings[host_key] != null }
 }
 
 module "srv" {
   source                         = "../libvirt/suse_manager"
   base_configuration             = module.base.configuration
   product_version                = var.product_version
+  image                          = lookup(local.images, "srv", null)
```
Comment (rjmateus) [inline modules/cucumber_testsuite/main.tf]: If the image is not defined we should use the default value for the variable which is: "default".
```suggestion
  image                          = lookup(local.images, "srv", "default")
```

File: modules/cucumber_testsuite/main.tf [hunk 2]
```diff
@@ -70,6 +72,7 @@ module "pxy" {
 
   base_configuration = module.base.configuration
   product_version    = var.product_version
+  image              = lookup(local.images, "pxy", null)
```
Comment (rjmateus) [inline modules/cucumber_testsuite/main.tf]: If the image is not defined we should use the default value for the variable which is: "default".
```suggestion
  image              = lookup(local.images, "pxy", "default")
```

File: modules/cucumber_testsuite/main.tf [hunk 3]
```diff
@@ -30,12 +29,15 @@ locals {
   host_key => lookup(var.host_settings[host_key], "mac", null) if var.host_settings[host_key] != null }
   additional_repos = { for host_key in local.hosts :
   host_key => lookup(var.host_settings[host_key], "additional_repos", {}) if var.host_settings[host_key] != null }
+  images = { for host_key in local.hosts :
+  host_key => lookup(var.host_settings[host_key], "image", null) if var.host_settings[host_key] != null }
```
Comment (rjmateus) [inline modules/cucumber_testsuite/main.tf]:
```suggestion
  host_key => lookup(var.host_settings[host_key], "image", "default") if var.host_settings[host_key] != null }
```
If the image value is not defined for the host, which default should we use? At least suse manager and suse manager proxy use "default" string as default value.

Comment (moio) [reply modules/cucumber_testsuite/main.tf]: Others will use different defaults though. I'd leave it as it is for now

**Pull request merged**